utils/flakiness-dashboard/processing/dashboard_compressed_v1.js

```diff
@@ -43,6 +43,16 @@ function compressReports(reports) {
           delete project.metadata.mode;
         if (project.metadata.platform && project.metadata.platform.toLowerCase() !== 'android')
           delete project.metadata.platform;
+        // Cleanup a bunch of data from report.
+        delete project.metadata['ci.link'];
+        delete project.metadata['revision.id'];
+        delete project.metadata['revision.author'];
+        delete project.metadata['revision.email'];
+        delete project.metadata['revision.subject'];
+        delete project.metadata['revision.timestamp'];
+        delete project.metadata['revision.link'];
+        delete project.metadata['timestamp'];
```

Comment:
We're likely to add more metadata in the future, so instead of deleting, let's do a safelisted approach.